configs/caas/cromwell.conf.ctmpl

```diff
@@ -302,20 +302,25 @@ languages {
     versions {
       "draft-2" {
         language-factory = "languages.wdl.draft2.WdlDraft2LanguageFactory"
+        config.caching {
+          enabled: true
+          ttl: 5 minutes
+          size: 100
+          concurrency: 9
+        }
       }
-      "draft-3" {
+      "1.0" {
```

Comment:
TOL: (and I doubt this is actually happening): if anyone is submitting to this Cromwell with the workflowLanguage param specified, they would need to update that when this changes

Reply:
We do not document a `draft-3` in the `workflowTypeVersion` on Swagger, so I'm not sure this is a concern